presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -0,0 +1,487 @@
+=================
+Presto properties
+=================
+
+This is a list and description of most important presto properties that may be used to tune Presto or alter it behavior when required.
+
+
+.. _tuning-pref-general:
+
+General properties
+------------------
+
+
+``distributed-joins-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Boolean``
+ * **Default value:** ``true``
+ * **Description:** Use hash distributed joins instead of broadcast joins. Distributed joins require redistributing both tables using a hash of the join key. This can be slower (sometimes substantially) than broadcast joins but allows much larger joins. Broadcast joins require that the tables on the right side of the join after filtering fit in memory on each node whereas distributed joins only need to fit in distributed memory across all nodes. This can also be specified on a per-query basis using the ``distributed_join`` session property.
+
+
+``redistribute-writes``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Boolean``
+ * **Default value:** ``true``
+ * **Description:** Force parallel distributed writes. Setting this property will cause the write operator to be distributed between nodes. This allows allows Presto to more efficiently utilize distributed storage across the cluster. It is particularly effective when executing a small number of huge queries. This can also be specified on a per-query basis using the ``redistribute_writes`` session property.
```

Comment:
Actually I think it doesn't work that way.

This turns on data redistribution between nodes before writing them to table writers. It's helpful if there is an significant data skew in `SOME_SKEWED_QUERY` when it is used in:

`INSERT INTO TABLE xyz SELECT SOME_SKEWED_QUERY`
`CREATE TABLE xyz AS SELECT SOME_SKEWED_QUERY`